Updated prenatal-shelter-suppl...

tekarrawilkinson opened the pull request:
Adding prenatal-shelter-suppl....

Review: https://brms-simulator.apps.silver.devops.gov.bc.ca/rule/6670cb5679140169e141e812?version=inReview&_=1728575654795

sbuhovac commented:
Rachel (@rachel0728) and I did Validation Testing for Pre-Natal Shelter Supplement rule. Here's our feedback:

- "Family Unit in Pay for month" input with selection "No" shows "No data" as a result, instead of not eligible.
- When "Shelter Costs" input is $0, the reason for not eligible says "Shelter costs not verified". It should show the same as when amount is $500 or less.

@tekarrawilkinson @prchristenson @Iliana777

tekarrawilkinson commented:
@sbuhovac 
I've updated the "Family Unit in Pay" to show ineligible when its false instead of no data.
If the shelter costs are $0 the reason will now say "No Shelter Costs" 

The reasons are imbedded in the eligibility calculation table and not the supplement output, so it will only provide a reason when the shelter costs are $0 (as its currently built). We don't have a reason output listed in all other rules, so I would like to discuss this more in our meeting this afternoon before I make any changes.

tekarrawilkinson commented:
@sbuhovac @rachel0728 updates to this rule were made in the walkthrough

shelterCosts must be >$500 and the reason field has been updated

sbuhovac commented:
Rachel (@rachel0728) and I did another round of validation testing and the updated rule includes our initial feedback. Thank you @tekarrawilkinson.

We don't have any further feedback, other than there is a missing Cancellation Date as an output. However, we realize Cancellation Date has not been built for any supplements that need it as that will require further discussion and decision making about it. 

Validation Testing can be passed over to @prchristenson and @Iliana777 .

Iliana777 commented:
@tekarrawilkinson @prchristenson @sbuhovac @rachel0728 
Ran two tests: 'Shelter costs are $500' - passed as expected but for some reason it is showing red (Tekarra, hoping you can help me fix this please) and 'Not single has dependent children' - passed as expected.

Some minor edits suggested:
1. When the family unit is not in pay, no reason is provided for the false result - suggest reason 'Family unit not in pay of assistance'
2. When shelter is under $500 - suggest reason 'Shelter costs are less than or equal to the unit one shelter rate' instead of less than or equal to $500 - I think it's better to identify that it is the unit one shelter rate (which may increase at any time) rather than $500 but I'm open to other's opinions on this one

rachel0728 commented:
> @tekarrawilkinson @prchristenson @sbuhovac @rachel0728 Ran two tests: 'Shelter costs are $500' - passed as expected but for some reason it is showing red (Tekarra, hoping you can help me fix this please) and 'Not single has dependent children' - passed as expected.
> 
> Some minor edits suggested:
> 
> 1. When the family unit is not in pay, no reason is provided for the false result - suggest reason 'Family unit not in pay of assistance'
> 2. When shelter is under $500 - suggest reason 'Shelter costs are less than or equal to the unit one shelter rate' instead of less than or equal to $500 - I think it's better to identify that it is the unit one shelter rate (which may increase at any time) rather than $500 but I'm open to other's opinions on this one

I would suggest "Shelter costs are less than or equal to the maximum shelter rate for a family unit of one"

tekarrawilkinson commented:
> > @tekarrawilkinson @prchristenson @sbuhovac @rachel0728 Ran two tests: 'Shelter costs are $500' - passed as expected but for some reason it is showing red (Tekarra, hoping you can help me fix this please) and 'Not single has dependent children' - passed as expected.
> > Some minor edits suggested:
> > 
> > 1. When the family unit is not in pay, no reason is provided for the false result - suggest reason 'Family unit not in pay of assistance'
> > 2. When shelter is under $500 - suggest reason 'Shelter costs are less than or equal to the unit one shelter rate' instead of less than or equal to $500 - I think it's better to identify that it is the unit one shelter rate (which may increase at any time) rather than $500 but I'm open to other's opinions on this one
> 
> I would suggest "Shelter costs are less than or equal to the maximum shelter rate for a family unit of one"


I have updated reason language to "Shelter costs are less than or equal to the maximum shelter rate for a family unit of one." We may have to wordsmith all of the reasons, but based on the conversation this morning with Jeremy, our focus during this pass won't be on making the reason codes perfect, but its good practice to include them. 

I've also added a reason for the family unit not in pay and added another scenario to capture that ineligibility.
